frontend/src/tasks/tasks.html

```diff
@@ -0,0 +1,63 @@
+<div class="p-4 space-y-6">
+    <h1 class="text-2xl font-bold text-gray-700">Task Overview</h1>
+    <!-- Task List -->
+    <div class="bg-white p-4 rounded-lg shadow">
+        <div class="mb-4">
+            <label class="block text-sm font-medium text-gray-700 mb-1">Filter by Date:</label>
+            <select v-model="selectedRange" @change="updateDateRange" class="border-gray-300 rounded-md shadow-sm w-full p-2">
+                <option v-for="option in dateFilters" :key="option.value" :value="option.value">
+                {{ option.label }}
+                </option>
+            </select>
+        </div>
+        <div class="mb-4">
+            <label class="block text-sm font-medium text-gray-700 mb-1">Filter by Status:</label>
+            <select v-model="selectedStatus" @change="getTasks" class="border-gray-300 rounded-md shadow-sm w-full p-2">
+                <option v-for="option in statusFilters" :key="option.value" :value="option.value">
+                {{ option.label }}
+                </option>
+            </select>
+        </div>
+        <div class="mb-6">
+            <button
+              @click="resetFilters"
+              class="w-full bg-gray-200 text-gray-700 hover:bg-gray-300 font-medium py-2 px-4 rounded-md transition"
+            >
+              Reset Filters
+            </button>
+        </div>
+          <!-- Summary Section -->
+        <div class="grid grid-cols-2 sm:grid-cols-4 gap-4">
+            <div class="bg-yellow-100 text-yellow-800 p-4 rounded-lg shadow">
+                <div class="text-sm">Scheduled</div>
+                <div class="text-2xl font-bold">{{pendingCount}}</div>
+            </div>
+            <div class="bg-green-100 text-green-800 p-4 rounded-lg shadow">
+                <div class="text-sm">Completed</div>
+                <div class="text-2xl font-bold">{{succeededCount}}</div>
+            </div>
+            <div class="bg-red-100 text-red-800 p-4 rounded-lg shadow">
+                <div class="text-sm">Failed</div>
+                <div class="text-2xl font-bold">{{failedCount}}</div>
+            </div>
+            <div class="bg-blue-100 text-blue-800 p-4 rounded-lg shadow">
+                <div class="text-sm">Cancelled</div>
+                <div class="text-2xl font-bold">{{cancelledCount}}</div>
+            </div>
+        </div>
+        <ul class="divide-y divide-gray-200">
+            <li v-for="task in tasks" :key="task.id" class="py-3 flex items-center justify-between">
+                <div>
+                    <div class="font-medium">{{ task.name }}</div>
```

Comment:
Just having a list of scheduled tasks isn't that useful, hard to see params. I think it would be more useful to create a layout similar to this one, which shows you how many tasks succeeded/failed/pending/cancelled, grouped by name.

![image](https://github.com/user-attachments/assets/31471812-67c1-4504-b2db-0494906bfce6)